metaworld/envs/mujoco/mujoco_env.py

```diff
@@ -106,9 +106,14 @@ def do_simulation(self, ctrl, n_frames=None):
         for _ in range(n_frames):
             self.sim.step()
 
-    def render(self, mode='human'):
+    def render(self, mode='human', res=(640, 480)):
```

Comment:
please make this a constructor param -- the `gym.Env` does not have any any arguments except for mode